cmd/traffic/logger.go

```diff
@@ -12,11 +12,11 @@ import (
 func makeBaseLogger() dlog.Logger {
 	logrusLogger := logrus.New()
 	logrusFormatter := &logrus.TextFormatter{
-		TimestampFormat: "2006-01-02 15:04:05",
+		TimestampFormat: "2006-01-02 15:04:05.0000",
 		FullTimestamp:   true,
 	}
 	logrusLogger.SetFormatter(logrusFormatter)
-	logrusLogger.SetReportCaller(false)
+	logrusLogger.SetReportCaller(true)
```

Comment:
Adding milliseconds is fine (and very much needed) but do we really need the caller here?

Reply:
I wanted to make it consistent with the daemon logs, plus I found the caller really useful for tracking down what `pkg/connpool` is doing -- are there any concerns around adding it?

Reply:
The reason I asked was actually for consistency. I don't see that this is enabled in the daemon logs.

Reply:
hmm, i see it enabled [here](https://github.com/telepresenceio/telepresence/blob/cb198902a5497b97e66bb31d71c7ab5d17ac791d/pkg/client/logging/initcontext.go#L30)